agent/consul/peering_backend.go

```diff
@@ -104,7 +104,7 @@ func (b *peeringBackend) Apply() peering.Apply {
 	return b.apply
 }
 
-func (b *peeringBackend) LeadershipMonitor() peering.LeadershipMonitor {
+func (b *peeringBackend) LeaderAddress() peering.LeaderAddress {
 	return b.monitor
```

Comment:
I think you missed this line? `peeringBackend` still has references to `leadershipMonitor`

Reply:
ah yes, thanks for that call out 💯